cypress/features/regression/experimental/search.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Search component
+  I want to change Search component properties
+
+  Background: Open Search component page
+    Given I open "Search" component page
+
+  @positive
+  Scenario Outline: Set placeholder to <placeholder>
+    When I set placeholder to "<placeholder>"
+    Then Search component placeholder is set to "<placeholder>"
+    Examples:
+      | placeholder             |
+      | Sample text             |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+      | <>                      |
+
+  @positive
+  Scenario Outline: Set inner text to <text> and threshold to <threshold>
```

Comment:
You check that `Then search icon has golden border`, change the name of `Scenario`

Reply:
fixed